person.rb

```diff
@@ -2,15 +2,15 @@
 
 class Person < Nameable
   attr_reader :id
-  attr_accessor :name, :age, :rental
+  attr_accessor :name, :age, :rentals
 
   def initialize(id = rand(1...100), name = 'Unknown', age = nil, parent_permission: true)
     super()
     @id = id
     @name = name
     @age = age
     @parent_permission = parent_permission
-    @rental = []
+    @rentals = []
```

Comment:
- [ ] While creating a `student` I still encounter this error
<img width="1127" alt="Screenshot 2023-07-28 at 18 16 47 1" src="https://github.com/enmanuelbayen/OOP-school-library/assets/44745855/8b9ee427-bb01-4c2d-b356-d1b10746eb82">

I suggest that you change it to
```ruby
 def initialize(age, name = "Unknown", parent_permission = true)
    @id = Random.rand(1..1000)
    @age = age
    @name = name
    @parent_permission = parent_permission
    @rentals = []
  end
```